src/runtime_src/CMakeLists.txt

```diff
@@ -61,6 +61,7 @@ set_target_properties(xilinxopencl PROPERTIES VERSION ${XRT_VERSION_STRING}
   SOVERSION ${XRT_SOVERSION})
 
 target_link_libraries(oclxdp
+  xdp
```

Comment:
doesn't xdp have to be added to dependencies for oclxdp some where around line 24?

Reply:
Yep good point Soren. I'll add that.